assets/css/amp-default.css

```diff
@@ -1,4 +1,4 @@
-.amp-wp-unknown-size img {
+.amp-wp-unknown-size .i-amphtml-replaced-content {
```

Comment:
It is illegal in AMP to use `i-amphtml-replaced-content` class names. See:

> Internal AMP class names prefixed with `-amp-` and `i-amp-` are disallowed in AMP HTML.

https://www.ampproject.org/docs/fundamentals/spec#classes

Reply:
Yes, think I even saw a todo note recently somewhere as well to remove these from CSS within style sanitizer and realized that it's illegal. Will look into which selector to use.

Reply:
Since tree shaking isn't done for attribute values, I suggest the way to fix this is to use an attribute selector like `[src]`.